bodhi/server/config.py

```diff
@@ -407,6 +407,12 @@ class BodhiConfig(dict):
         'initial_bug_msg': {
             'value': '%s has been submitted as an update to %s. %s',
             'validator': unicode},
+        'greenwave_api_url': {
+            'value': 'https://greenwave.fedoraproject.org/api/v1.0',
+            'validator': unicode},
+        'greenwave_enabled': {
+            'value': False,
+            'validator': _validate_bool},
```

Comment:
I had a talk with @ralphbean today about the confusion around the existing CI code that @pypingou wrote and the new plans for adding Greenwave support to Bodhi. We came to a conclusion together that we should try to resolve the differences in this patch rather than adding another way to gate updates (because without this patch there are already two ways to gate - ci_status and the code that blocks updates based on test results).

All that to say - there is already a ```ci.enabled``` setting - rather than having a ```greenwave_enabled``` setting, why not just make ```ci_enabled``` mean that we will try to talk to Greenwave? Later in this pull request I see that you have replaced usages of ```ci_enabled``` anyway, so it probably doesn't make sense to keep both settings. I prefer the ```ci_enabled``` name just for being more generic and meaningful to admins that might not be familiar with Greenwave but will know what CI means.